Src/AutoFixtureUnitTest/Kernel/CriterionTests.cs

```diff
@@ -94,9 +94,9 @@ public void SutEqualsIdenticalValue()
         [InlineData(typeof(Version))]
         public void SutDoesNotEqualAnyObject(object other)
         {
-            var sut = new Criterion<PlatformID>(
-                PlatformID.Unix,
-                new DelegatingEqualityComparer<PlatformID>());
+            var sut = new Criterion<ConsoleColor>(
+                ConsoleColor.Cyan,
+                new DelegatingEqualityComparer<ConsoleColor>());
             var actual = sut.Equals(other);
```

Comment:
I guess, all these changes are done because some objects aren't available in .NET Standard?